MaterialComponent.js

```diff
@@ -53,23 +57,30 @@ export default class MaterialComponent extends Component {
     }
     return classText;
   }
+
   // Components must implement this method for their specific DOM structure
   materialDom(props) {
     return h("div", Object.assign({}, props), props.children);
   }
+
   render() {
     this.buildClassName();
     // Fetch a VNode
     const componentProps = this.props;
+    const userDefinedClasses =
+      componentProps.className || componentProps.class || "";
     if (componentProps.class) {
```

Comment:
If we check for whether `class` exists why do we also remove `className`? Shouldn't it be put into its own if statement?

```JavaScript
// Short example
if (componentProps.class) delete componentProps.class;
if (componentProps.className) delete componentProps.className;
```